README.md

```diff
@@ -185,79 +185,105 @@ import (
 
 func Test_Something_With_Docker(t *testing.T) {
   opts := &docker.DockerClusterOptions{
-    ImageRepo: "hashicorp/vault",
+    ImageRepo: "hashicorp/vault", // or "hashicorp/vault-enterprise"
```

Comment:
Do we need this comment if we have a whole separate example below for enterprise which includes the licensing config?

Reply:
I'm kind of trying to cater to two different audiences here, external and internal.  For internal, we're generally going to want to be testing using locally built binaries (on dev machine or in CI), and so the repo is irrelevant.  For external, they're going to be testing their integration with vault rather than testing their own changes to vault, so it may be easier/faster for them to just use an upstream docker image.